mix.exs

```diff
@@ -20,13 +20,12 @@ defmodule ExVCR.Mixfile do
   end
 
   def application do
-    [applications: [:meck, :exactor, :exjsx], mod: {ExVCR.Application, []}]
+    [applications: [:meck, :exjsx], mod: {ExVCR.Application, []}]
   end
 
   def deps do
     [
       {:meck, "~> 0.8"},
-      {:exactor, "~> 2.2"},
```

Comment:
this should also be removed from the mix.lock file

`mix deps.clean --unused --unlock`